### ISSUE TITLE
Fix structure of search page

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#242]
There will need to be some restructuring of the L&C search page similar to the changes made in Orchid (to address "Ensure elements with an ARIA role that require parent roles are contained by them"), but I'm not quite sure how best this should be done. Adding here as comprehensive task but it looks like filters and facets are a particular issue.

**Comment by nichgray** [comment]
I think I fixed the thing that was causing this issue, but two more popped up. Will create those separately.
